tests/unit/index_tests.py

```diff
@@ -584,5 +585,134 @@ def test_delete_disabled(self):
             'Deleting the \"special\" index is not allowed.'
         )
 
+@unittest.skipUnless(
+    os.environ.get('RUN_CLOUDANT_TESTS') is not None,
```

Comment:
This skip code is used in other test modules.  I would like to handle this change as a separate issue.